### PR TITLE
fix: update cypress-axe

### PR DESCRIPTION
**Pull request by dylanb**

### CODE REVIEW

File: cypress/support/index.js [hunk 1]
```diff
@@ -23,11 +23,11 @@ import 'cypress-axe';
 const { axe } = JSON.parse(Cypress.env('CONFIG'));
 
 before(() => {
-  cy.visit('http://localhost:1234');
+  cy.visit('http://localhost:1235');
```
Comment (dylanb) [inline cypress/support/index.js]: so it does not conflict with walnut

File: .babelrc [hunk 2]
```diff
@@ -3,6 +3,5 @@
     "@babel/env", "@babel/react"
   ],
   "plugins": [
-    "@babel/plugin-proposal-class-properties"
```
Comment (dylanb) [inline .babelrc:6]: Not sure why this was removed, could it be yarn?

File: cypress/plugins/index.js [hunk 3]
```diff
@@ -17,6 +17,11 @@ module.exports = (on, config) => {
       runOnly: {
         type: 'tag',
         values: ['wcag2a', 'wcag2aa']
+      },
```
Comment (dylanb) [inline cypress/plugins/index.js:20]: stupid rule kept failing everywhere

File: package.json [hunk 4]
```diff
@@ -76,10 +76,10 @@
     "@babel/preset-stage-0": "^7.0.0",
     "ally.js": "^1.4.1",
     "autoprefixer": "^9.5.1",
-    "axe-core": "^3.2.2",
+    "axe-core": "^4.1.2",
     "babel-eslint": "^10.0.1",
-    "cypress": "^3.2.0",
-    "cypress-axe": "^0.4.0",
+    "cypress": "^5.6.0",
```
Comment (dylanb) [inline package.json:81]: updated to support Saucelabs

File: .babelrc (hunk 2 above)
Comment (schne324) [reply .babelrc:6]: I think we need this still. It allows us to:

```js
class Foo extends React.component {
  myCoolClassMethod = () => console.log('now you dont have to `bind` this method in contructor')
}
```

**Review by schne324 — APPROVED** [review]

**Pull request merged**